src/gql/queries/get-mainline-commits-for-history.graphql

```diff
@@ -2,7 +2,10 @@ query MainlineCommitsForHistory(
   $mainlineCommitsOptions: MainlineCommitsOptions!
   $buildVariantOptions: BuildVariantOptions!
 ) {
-  mainlineCommits(options: $mainlineCommitsOptions) {
+  mainlineCommits(
```

Comment:
Just to confirm based on the behavior in this PR, the collapsing should only happen when visiting a page and not when applying any task/variant filters? If so then maybe the description in the ticket [EVG-16567](https://jira.mongodb.org/browse/EVG-16567) should be updated